src/gerbil/runtime/system.ss

```diff
@@ -13,7 +13,7 @@ namespace: #f
   [["Gerbil" :: (gerbil-version-string)]
    ["Gambit" :: (system-version-string)]])
 
-(defmutable build-manifest gerbil-system-manifest : :list)
+(defmutable* build-manifest gerbil-system-manifest : :list)
```

Comment:
Why use defmutable* rather than defmutable?

And if so, you must update std/cli/multicall.ss (as found with git grep build-manifest).

Reply:
we cannot export macros from the runtime through the prelude interface, as I have explained to you.